CONTRIB.rst

```diff
@@ -84,6 +84,7 @@ please open an issue on GitHub or mention it on the mailing list.
 - David Weisman <david.weisman at domain acm.org>
 - David Winter <https://github.com/dwinter>
 - Devang Thakkar <https://github.com/devangthakkar>
+- Deepak Khatri <https://github.com/deepaklorkhatri007>
```

Comment:
Can you move name before Devang please (simple sorting - this failed the automated check - see the comment at the top of the file).

Reply:
Really sorry about that, I have tested the Tutorial.pdf files and created a table on #2068 please take a look.

Reply:
Thank you - for both points.

Reply:
@peterjc please don't say thank you, it's a dream come true really, I know python and I want to contribute in python code too. is there anything I should keep in mind for that?

Reply:
Maybe have a browse of the open issues and pull requests, and see if anything sparks your interest?

I would also suggest installing flake8 and the pre-commit hook (see https://github.com/biopython/biopython/blob/master/CONTRIBUTING.rst for details) if you start working on our Python files.

Reply:
@peterjc got it :100:   :)